sylt/src/formatter.rs

```diff
@@ -638,6 +638,6 @@ fn format_module(module: Module) -> Result<String, fmt::Error> {
 }
 
 pub fn format(file: &Path) -> Result<String, Vec<Error>> {
-    let mut tree = sylt_parser::tree(file, crate::read_file)?;
+    let mut tree = sylt_parser::tree(file, crate::read_file, false)?;
```

Comment:
I feel like this is a bit arbitrary? Maybe we should mark the imports in the syntax-tree as being default imported? If this flag should exist, it should be hooked up to the argument-parsing aswell - but maybe you've just not had time for that.